Makefile

```diff
@@ -190,10 +143,11 @@ demo_msvc64:  flexlink.exe flexdll_msvc64.obj flexdll_initer_msvc64.obj
 	(cd test && $(MSVC64_PREFIX) $(MAKE) clean demo CHAIN=msvc64 CC="$(MSVCC64)" O=obj)
 
 distclean: clean
-	rm -f Makefile.winsdk
+	rm -f Makefile.config config.log config.status version.ml
+	rm -rf autom4te.cache
 
 clean:
-	rm -f *.obj *.o *.lib *.a *.exe *.opt *.cmx *.dll *.exp *.cmi *.cmo *~ version.res version.ml
+	rm -f *.obj *.o *.lib *.a *.exe *.opt *.cmx *.dll *.exp *.cmi *.cmo *~ version.res
```

Comment:
`version.ml` is generated by `configure`, so should be removed in `distclean` only.